src/Gtransfo.cc

```diff
@@ -263,22 +263,18 @@ class GtransfoInverse : public Gtransfo {
   GtransfoInverse(const GtransfoInverse&);
 
   //! Overload the "generic routine"
-  Gtransfo* RoughInverse(const Frame &Region) const
+  Gtransfo* RoughInverse(const Frame &) const
```

Comment:
Should this be `roughInverse` (lowercase R)? Similarly for `InverseTransfo` below? It's a bit hard to tell from the diff.